crates/fuel-streams-publisher/src/payloads/mod.rs

```diff
@@ -0,0 +1,8 @@
+pub mod blocks;
+pub mod identifiers;
+pub mod inputs;
+pub mod logs;
+pub mod outputs;
+pub mod receipts;
+pub mod transactions;
+pub mod utxos;
```

Comment:
Since only the publisher consumes these, this should reside in the `publisher` module

Reply:
As a sub-sub module of course